notebooks/5-stellar-flares/interm_notebook_5_classify.ipynb

```diff
@@ -0,0 +1,1428 @@
+{
```

Comment:
<p>Could we select a different combination of targets / sectors, such that at least one of our examples contains a flare? Given that <code>max(flare_probability) </code>in the above is ~0.125, the plot might actually look quite normal (assuming the classifier uses &gt; 0.5 as the cut-off threshold for a flare).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>